gui/qt/main_window.py

```diff
@@ -389,7 +389,7 @@ def open_wallet(self):
         wallet_folder = self.get_wallet_folder()
         filename, __ = QFileDialog.getOpenFileName(self, "Select your wallet file", wallet_folder)
         if not filename:
-            return
+            self.show_error('Error opening the file. Please check if the file is accessible.')
```

Comment:
Are you sure that the `return` can be safely removed? Obviously, removing it means that the line below will execute. I can't say for certain if it's ok, because I am not even sure why the "crash" even occurs...

Reply:
IIRC getOpenFileName returns None if the user cancels the dialog.
in that case it is unsafe to remove